bmq-sdk/src/test/java/com/bloomberg/bmq/it/BrokerSessionIT.java

```diff
@@ -235,14 +234,13 @@ public void openConfigureCloseQueueAsyncTest() {
         logger.info("BEGIN Testing BrokerSessionIT openConfigureCloseQueueAsyncTest.");
         logger.info("=================================================================");
 
-        final int TESTED_PORT = SystemUtil.getEphemeralPort();
         final Semaphore testSema = new Semaphore(0);
-        BmqBrokerSimulator server = new BmqBrokerSimulator(TESTED_PORT, Mode.BMQ_AUTO_MODE);
+        BmqBrokerSimulator server = new BmqBrokerSimulator(Mode.BMQ_AUTO_MODE);
         server.start();
 
         SessionOptions sessionOptions =
                 SessionOptions.builder()
-                        .setBrokerUri(URI.create("tcp://localhost:" + TESTED_PORT))
+                        .setBrokerUri(URI.create("tcp://localhost:" + server.getPort()))
```

Comment:
Don't know if it's worth it to make `server.getURI()` or just `server.getSessionOptions()` to simplify even more.

Reply:
Like the idea of having `server.getURI()`

Reply:
Also like the `server.getSessionOptions()` method.
Besides, if the options should be updated, now we have `SessionOptions::build(SessionOptions)` for that

Reply:
Tried `server.getSessionOptions()`, but we have frequent `setHostMonitor` which makes things not as good as I hoped. Used `getURI()` instead.